common/lib/xmodule/xmodule/course_module.py

```diff
@@ -34,6 +34,15 @@
 CATALOG_VISIBILITY_NONE = "none"
 
 
+def get_proctoring_list():
+    from django.conf import settings
```

Comment:
`from django.conf import settings`
XModules have to know nothing about `Django`